packages/react-instantsearch/src/components/List.js

```diff
@@ -80,14 +80,14 @@ class List extends Component {
   renderShowMore() {
     const {showMore, translate, applyTheme} = this.props;
     const {extended} = this.state;
-
+    const disabled = this.props.limitMin >= this.props.items.length;
     if (!showMore) {
       return null;
     }
 
     return (
       <button
-        {...applyTheme('showMore', 'showMore')}
+        {...applyTheme('showMore', 'showMore', disabled && 'showMoreDisabled')}
```

Comment:
I would compute that in an outer variable that I would then use :) I think that it creates a better readability. Also using && will propagate the value `false` which is probably not what we want.

```jsx
    if (!showMore) {
        return null;
    }

    const disabled = this.props.limitMin >= this.props.items.length;
    const theme = disabled ?
        applyTheme('showMore', 'showMore', 'showMoreDisabled'):
        applyTheme('showMore', 'showMore');
      
    return (
        <button {...theme} >
        </button>
    );
```